File: Examples/js/player.js
"use strict";

var player = {
	score : undefined,
	timer : undefined,
	
	init : function(score, timer)
	{
		this.score = score;
		this.timer = timer;
		
		console.log("score: " + this.score);
		console.log("timer: " + this.timer);
	}
};

File: src/Assets/Scripts/Player.cs
using UnityEngine;
using System.Collections;

public class player : MonoBehaviour {

	//Score of current Player
	private int _score;

	public int score{

		get{ return _score;}
		set{ _score = value;}

	}
	//Timer of current Player
	private Time _timer;



	//Name of current Player
	private string _name;

	public string name{

		get{ return _name;}
		set{ _name = value;}

	}



	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
	
	}
}
